### PR TITLE
doc: quick_start: edits from testing

### PR DESCRIPTION
**Pull request by greg-fer**
Applied edits from testing the Quick Start app with @ketile.
Added information about how Developing with pages relate to other guides
in Application development.
Broke down testing steps for samples supported in Quick Start into
passive behavior that can be observed after programming using the app
and steps that require additional applications.

### CODE REVIEW

**Review by ketile — APPROVED** [review]
Looks good to me. Maybe double check that any references to LEDs or Buttons on the DK are 0 indexed for 54 Series (unlike earlier kits).

File: samples/bluetooth/peripheral_lbs/README.rst [hunk 1]
```diff
@@ -134,13 +134,15 @@ For example, when building on the command line, you can add the option as follow
 Testing
 =======
 
-After programming the sample to your dongle or development kit, test it by performing the following steps:
+After programming the sample to your dongle or development kit, the sample starts the advertising loop and one of the LEDs starts blinking to indicate this (see `User interface`_ for details).
```
Comment (annwoj) [inline samples/bluetooth/peripheral_lbs/README.rst:137]:
```suggestion
After programming the sample to your dongle or development kit, one of the LEDs starts blinking, indicating the advertising loop is active (see `User interface`_ for details).
```

File: samples/bluetooth/peripheral_uart/README.rst [hunk 2]
```diff
@@ -226,10 +227,44 @@ After programming the sample to your development kit, complete the following ste
...
       #. Observe that the text "Starting Nordic UART service example" is printed on the COM listener running on the computer.
+
+   .. group-tab:: nRF54 DKs
+
+      .. note::
+          |nrf54_buttons_leds_numbering|
+
+      1. Connect the device to the computer to access UART 0.
+         If you use a development kit, UART 0 is forwarded as a COM port (Windows) or ttyACM device (Linux) after you connect the development kit over USB.
+         If you use Thingy:53, you must attach the debug board and connect an external USB to UART converter to it.
```
Comment (annwoj) [inline samples/bluetooth/peripheral_uart/README.rst:241]:
```suggestion
         If you use Thingy:53, you must attach the debug board and connect it to an external USB to UART converter.
```

File: samples/bluetooth/peripheral_uart/README.rst [hunk 3]
```diff
@@ -226,10 +227,44 @@ After programming the sample to your development kit, complete the following ste
...
+
+To perform the test, complete the following steps:
+
+.. tabs::
+
+   .. group-tab:: nRF21, nRF52 and nRF53 DKs
+
+      1. Connect the device to the computer to access UART 0.
+         If you use a development kit, UART 0 is forwarded as a COM port (Windows) or ttyACM device (Linux) after you connect the development kit over USB.
+         If you use Thingy:53, you must attach the debug board and connect an external USB to UART converter to it.
```
Comment (annwoj) [inline samples/bluetooth/peripheral_uart/README.rst:263]:
```suggestion
         If you use Thingy:53, you must attach the debug board and connect it to an external USB to UART converter.
```

**Review by annwoj — COMMENTED** [review]
minor suggestion

**Review by annwoj — APPROVED** [review]

**Pull request merged**